src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let stateClone = { ...state };
+
+  for (const action of actions) {
+    if (action.type === 'addProperties') {
```

Comment:
Good job, but better use switch/case as u can read in checklist!
Good luck

Reply:
thank you! looked more understandable for me though :)